pkg/config/config.go

```diff
@@ -8,19 +8,22 @@ import (
 	"github.com/evalphobia/logrus_sentry"
 	raven "github.com/getsentry/raven-go"
 	newrelic "github.com/newrelic/go-agent"
+	"github.com/quipo/statsd"
```

Comment:
For new dependencies, can you also add it to Gopkg.toml and check in the vendor for faster build time?

Reply:
sure, I forgot to regenerate the gopkg file, will do!